giskard-frontend/src/views/main/project/PredictionExplanations.vue

```diff
@@ -154,12 +155,34 @@ export default class PredictionExplanations extends Vue {
   }
 
   get chartOptionsMultiClassification() {
-    const firstExplanations =
-      this.fullExplanations[Object.keys(this.fullExplanations!)[0]];
+    const labels = Object.keys(this.fullExplanations!);
+    const firstExplanations = this.fullExplanations[labels[0]];
+    const topFeatures = Object.keys(firstExplanations!);
+    // Compute the sum of SHAP explanations by feature
+    const explanationSum: Array<number> = topFeatures.map(
+      feature => labels.map(
+        label => this.fullExplanations[label][feature]
+      ).reduce((a, b) => a + b, 0));
+    // Create an object with key Feature and value the sum of SHAP explanations
+    const explanationSumByFeature: object = {};
+    topFeatures.forEach((element, index) => {
+      explanationSumByFeature[element] = explanationSum[index];
+    });
```

Comment:
Sure, but I wonder if this version is not less readable? Also, are you 100% sure this is the same formula as the code I wrote? Just asking, since I spent over 3 hours to craft the formula and make sure it was correct.